src/components/ArticlePage.jsx

```diff
@@ -8,14 +8,22 @@ const ArticlePage = () => {
     const {article_id} = useParams()
     const [article, setArticle] = useState()
     const [isLoading, setIsLoading] = useState(true)
+    const [votes, setVotes] = useState()
 
     useEffect(() => {
         getArticleById(article_id).then((res) => {
             setArticle(res.data.article)
+            setVotes(res.data.article.votes)
             setIsLoading(false)
         })
     }, [])
 
+    function handleVote(article_id, incVote) {
```

Comment:
Great that you've got one function able to handle both an upvote and a downvote, nice and DRY! ☂️